### DISCUSSION TITLE
Unable to filter by empty M2M in API or insights

### DISCUSSION THREAD

**Discussion by krazyjakee** [directus/directus#11039] [Archive]
Simple, I would like to find all instances of a collection where an M2M field is empty.

I have tried...
```
query {
      functions(filter: { parent_functions: { id: { _null: true } } })   {
        parent_functions {
          related_functions_id {
            id
            name
          }
        }
      }
    }
```
But the response is an empty array. Can't seem to find an alternative in the docs.
This issue also affects insights.

**Comment by oreilles** [comment]
I believe this unfortunately is a known limitation of the API (see https://github.com/directus/directus/issues/5996) that is being worked on.

**Reply by krazyjakee** [reply]
Thanks, I added my 2 cents to that issue.
